src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape,
+  color: Color,
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0
+     || this.b <= 0
+     || this.c <= 0) {
+      throw new Error('incorrect figure parameters');
+    }
+
+    const longestSide = Math.max(a, b, c);
+
+    if (longestSide >= (a + b + c - longestSide)) {
+      throw new Error('result does not match desired parameters');
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const perimeter = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(perimeter
+      * (perimeter - this.a)
+      * (perimeter - this.b)
+      * (perimeter - this.c));
 
+    return Math.round(area * 100) / 100;
```

Comment:
Optional: you round with the same logic twice over the code. You also could create a separate helper function - `rounder` for example for this purpose.